listenstore/listenstore/listenstore.py

```diff
@@ -119,16 +119,18 @@ def fetch_listens_from_storage(self, user_id, from_id, to_id, limit, order, prec
                      , listen_json
                  WHERE listen.id = listen_json.id
                    AND user_id = :user_id
-                   AND extract(epoch from ts) > :from_id
-                   AND extract(epoch from ts) < :to_id
-              ORDER BY extract(epoch from ts) """ + ORDER_TEXT[order] + """
+                   AND ts > :from_ts
+                   AND ts < :to_ts
+              ORDER BY ts """ + ORDER_TEXT[order] + """
                  LIMIT :limit
             """), {
                 'user_id': user_id,
-                'from_id': from_id,
-                'to_id': to_id,
+                'from_ts': datetime.utcfromtimestamp(from_ts),
+                'to_ts': datetime.utcfromtimestamp(to_ts),
```

Comment:
Does it make sense for this value to be a float? What does a timestamp of `1469203685.5` mean?